### DISCUSSION TITLE
Opus Titász átállás

### DISCUSSION THREAD

**Discussion by Deadfox531** [ZsBT/hass-w1000-portal#24] [Q&A] [answered]
Szia

Az EON-tól átveszi a távleolvasást az Opus Titász, nekem már az eon felületén nincs adat Október 1-óta. Az integráció képes a Titász oldaláról lekérdezni az adatokat?

**Comment by ZsBT** [comment]
Szia!
Én is megkaptam az értesítőt, de én úgy értelmeztem, hogy csak a korábbi Titász ügyfeleket érinti a változás. Nálam az EON w1000-ben még látszik tegnapi adat. Én volt ELMŰs fajta ügyfél vagyok. Persze ez nem jelenti azt, hogy nem fogják az összes áramfogyasztót Lölő alá bedarálni hamarosan.
Ha a Titász nem a [cseh szoftvercég w1000](https://www.landisgyr.cz/product/w1000/) portál nevű portékáját használja, akkor nem fog működni ez az integráció, újat kell készíteni.


Mások hogy látják az adataikat?

**Reply by tomykacsa** [reply]
Sziasztok!
https://hu.wikipedia.org/wiki/Elosztóhálózat-üzemeltető
https://hu.wikipedia.org/wiki/Elosztóhálózat-üzemeltető#/media/Fájl_Magyarországi_áramhálózati_cégek_2021_vége.png

https://energiadiszkont.hu/aktualitasok/pod-azonosito/

Az a lényeg, hogy hol van a fogyasztási hely ahhoz a hálózathoz tartozik és a mérésért a Hálózat a felelős!
És annak a hálózatnak fizeted a Rendszerhasználati díjat!
A villany az egy termék azt veheted akármilyen Energiakereskedőtől, vagy lehetsz Lakossági (ami már MVM)

Hálózat:

E.ON: ÉDÁSZ; DÉDÁSZ; ELMŰ (E.ON Távleolvasási Portál)

OPUS: TITÁSZ (Új OPUS Távleolvasási Portál)

MVM: ÉMÁSZ; DÉMÁSZ

**Comment by Deadfox531** [comment, marked as answer]
![Screenshot_2023-10-08-09-38-48-756_com android chrome](https://github.com/ZsBT/hass-w1000-portal/assets/65756216/1405c6b0-8287-459c-bc12-efc6a6a7b1ff)

Ez kaptam az eon w1000 portálon.

Már lehet regisztrálni az új Opus Titász leolvasó oldalra, bár még vannak vele problémák.
https://tavleolvasasiportal.opusenergetika.hu/login
